stac_fastapi/api/stac_fastapi/api/routes.py

```diff
@@ -37,7 +35,6 @@ async def run(*args, **kwargs):
 def create_async_endpoint(
     func: Callable,
     request_model: Union[Type[APIRequest], Type[BaseModel], Dict],
-    response_class: Type[Response] = JSONResponse,
```

Comment:
This could be considered as a breaking change, so we could make it optional and raise a warning